### PR TITLE
Added support for deleting a sending domain

### LINKED ISSUE #110: Add support for deleting a sending domain

### PR DESCRIPTION
**Pull request by aydrian**
Closes #110

### CODE REVIEW

File: examples/sendingDomains/delete_sendingDomain.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var key = 'YOURAPIKEY'
+  , SparkPost = require('sparkpost')
+  , client = new SparkPost(key);
+
+client.sendingDomains['delete']('example1.com', function(err, data) {
```
Comment (colestrode) [inline examples/sendingDomains/delete_sendingDomain.js]: We don't need to quote `delete` in Node. If this is failing linting, we should update the jshint to es3:false

I see this in a few places in this PR, I'll just comment this once.

Comment (aydrian) [inline examples/sendingDomains/delete_sendingDomain.js]: It is everywhere. I'll fix it in this PR. I'll create an issue to clean that up as part of v2.0.0

**Comment by colestrode** [comment]
Looks great. I had one comment about quoting the delete keyword, once that's resolved we can :shipit:

File: examples/sendingDomains/delete_sendingDomain.js (hunk 1 above)
Comment (colestrode) [inline examples/sendingDomains/delete_sendingDomain.js]: :+1:

**Comment by colestrode** [comment]
:shipit:

**Pull request merged**